flight/mcu/lib/teensyShared/Abort.cpp

```diff
@@ -0,0 +1,11 @@
+#include "Abort.h"
+
+bool is_aborted = false;
+
+void startAbort() {
```

Comment:
I thought that once we get to staging, we might want to add more mechanism around aborting, so a file would be good for that. And since aborting is something which disparate parts of the code will want to read and write, I didn't think it prudent to put its logic into some other piece of code.